eclair-core/src/main/scala/fr/acinq/eclair/PimpKamon.scala

```diff
@@ -34,10 +35,19 @@ object KamonExt {
     }
   }
 
+  def time[T](timer: Timer)(f: => T): T = {
+    val started = timer.start()
```

Comment:
I think `started` is a better name but you should have changed the method above too for consistency